ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Feature Request
+description: Request an enhancement or new feature
+labels:
+  - enhancement
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: >-
```

Comment:
Will we add acceptance criteria here, in its own section, or is the expectation that these feature requests will be converted to stores with that info?